Makefile

```diff
@@ -3,9 +3,9 @@
 
 MF_DOCKER_IMAGE_NAME_PREFIX ?= mainflux
 BUILD_DIR = build
-SERVICES = clients users things http coap ws lora influxdb-writer influxdb-reader mongodb-writer \
+SERVICES = clients things http coap ws lora influxdb-writer influxdb-reader mongodb-writer \
```

Comment:
You can still use the same name for the service: `users` because we'll have `things` clients, so we avoid conflicting names. And you removed old `users` anyways.